src/System.IdentityModel.Tokens.Jwt/JwtSecurityToken.cs

```diff
@@ -212,7 +212,7 @@ public JwtSecurityToken(string issuer = null, string audience = null, IEnumerabl
         /// <summary>
         /// Gets the 'value' of the 'actor' claim { actort, 'value' }.
         /// </summary>
-        /// <remarks>If the 'actor' claim is not found, null is returned.</remarks> 
+        /// <remarks>If the 'actor' claim is not found, an empty string is returned.</remarks> 
```

Comment:
an empty string is logically different than null.
Sometimes the existence of a value may open a gate, should we consider returning null?

Reply:
My concern about returning null would be that changing it from an empty string to null is considered a breaking change (for existing properties like `Issuer` and `Actor`). We could do this for new properties that are being added but that would be inconsistent behavior....